examples/simple-non-wasm/src/main.rs

```diff
@@ -26,119 +25,35 @@ pub struct Args {
 }
 
 // Load graphql schema
-graphql_schema!("simple_handler", "schema/counter.graphql");
+graphql_schema!("counter", "schema/counter.graphql");
 
-// Object being indexed
-#[derive(Deserialize, Serialize, Debug, Eq, PartialEq)]
-pub struct CountEvent {
-    count: u64,
-    timestamp: u64,
-}
-
-// Implement Tokenizable for object being indexed
-impl Tokenizable for CountEvent {
-    fn from_token(token: Token) -> Result<Self, InvalidOutputType> {
-        let arr: Vec<u8> = Vec::from_token(token)?;
-        let count = u64::from_be_bytes(arr[..8].try_into().unwrap());
-        let timestamp = u64::from_be_bytes(arr[8..].try_into().unwrap());
-
-        Ok(Self { count, timestamp })
-    }
-
-    fn into_token(self) -> Token {
-        Token::Array(vec![Token::U64(self.count), Token::U64(self.timestamp)])
-    }
-}
-
-// Implement Entity trait for object being indexed
-impl Entity for CountEvent {
-    const TYPE_ID: u64 = 0;
-    fn from_row(row: Vec<FtColumn>) -> Self {
-        let count = match (&row[0]).to_owned() {
-            FtColumn::Bytes8(x) => u64::from_le_bytes(*x),
-            _ => panic!("Invalid column type"),
-        };
-
-        let timestamp = match (&row[1]).to_owned() {
-            FtColumn::Bytes8(x) => u64::from_le_bytes(*x),
-            _ => panic!("Invalid column type"),
-        };
-
-        Self { count, timestamp }
-    }
+// Load structs from abigen
+abigen!(
+    Counter,
+    "examples/simple-non-wasm/programs/counter/out/debug/counter-abi.json"
+);
 
-    fn to_row(&self) -> Vec<FtColumn> {
-        vec![FtColumn::UInt8(self.count), FtColumn::UInt8(self.timestamp)]
-    }
+fn count_handler(data: Vec<u8>) -> Result<Option<Count>> {
```

Comment:
- After the discussion [here](https://fuellabs.slack.com/archives/C02CX4JVCSJ/p1655834736318249) I feel like this is as simple as I can get a handler at this point. 
- The user will just have to know what `ParamType` they need
- If anyone has a way to automagically do this, I'm open to ideas